packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,9 +1,20 @@
+## 42.7.0 - March 15, 2018
+### Added
+#### salesforcedx-vscode-core
+
+* Add demo mode for VS Code ([PR #335](https://github.com/forcedotcom/salesforcedx-vscode/pull/335))
```

Comment:
Added this since there are new commands available on palette (SFDX: Log Out from All Authorized Orgs)